### PR TITLE
Move the extension pack to .NET 8

### PR DESCRIPTION
**Pull request by mavasani**
Also bump the version of the extension and update the changelog

Similar to the prior changes #393 and #394 for .NET 7 move

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "vscode-dotnet-pack",
-  "version": "1.0.12",
+  "version": "1.0.13",
```
Comment (mavasani) [inline package.json:3]: @colombod Do we also need to update the minimum required vscode version below, just like we did last time when we moved to .NET 7 SDK as the jupyter extension needed vscode 1.74 or greater:

```json
"engines": {
    "vscode": "^1.74.0"
  },
```

Comment (mavasani) [reply package.json:3]: @arkalyanms I do see the CDK requires vscode version 1.80 or greater, so probably it makes sense to update the require vscode for this extension pack to the same?

Comment (colombod) [reply package.json:3]: i would fix the minimum version to 1.83 as this is the version that integrates some api change in vscode that polyglot notebook relies upon.

**Comment by mavasani** [comment]
Thanks @colombod - addressed your feedback. Can you please approve if the changes look good to you?

**Comment by colombod** [comment]
this is the PR in .NET Interactive to release .NET 8 support https://github.com/dotnet/interactive/pull/3336

**Review by colombod — APPROVED** [review]
Looks good to me!

**Pull request merged**